app/src/main/java/jp/co/cyberagent/stf/Service.java

```diff
@@ -455,14 +455,18 @@ public void run() {
                         String currentUsbState = "";
                         String currentAdbState = "";
                         BufferedReader adbdStateReader = new BufferedReader(new InputStreamReader(process.getInputStream()));
+
                         for (String line = adbdStateReader.readLine(); line != null; line = adbdStateReader.readLine()) {
                             if (line.contains("Current Functions:") || line.contains("mCurrentFunctions:")) {
                                 currentAdbState = line.split(":").length == 2 ? line.split(":")[1] : "";
                             } else if (line.contains("Kernel state:")) {
                                 currentUsbState = line.split(":").length == 2 ? line.split(":")[1] : "";
+                            }else if (line.toLowerCase().contains("kernel_state")) {
```

Comment:
```suggestion
                            } else if (line.toLowerCase().contains("kernel_state")) {
```

LGTM, but the sign-off is missing. You can use e.g. `git commit -s`